qiskit_braket_provider/providers/adapter.py

```diff
@@ -173,6 +175,9 @@
     "z": qiskit_gates.ZGate(),
     "zz": qiskit_gates.RZZGate(Parameter("theta")),
     "ecr": qiskit_gates.ECRGate(),
+    "prx": qiskit_gates.RGate(
+        Parameter("theta") / (2 * pi), Parameter("phi") / (2 * pi)
+    ),
```

Comment:
Are you sure that this unit conversion is necessary here? The documentation for `RXGate` and `RGate` appears similar to me - I would have expected that both are in radians and this division by 2*pi isn't needed.